Doc/library/select.rst

```diff
@@ -475,13 +475,14 @@ Kqueue Objects
    Create a kqueue object from a given file descriptor.
 
 
-.. method:: kqueue.control(changelist, max_events[, timeout=None]) -> eventlist
+.. method:: kqueue.control(changelist, max_events[, timeout]) -> eventlist
```

Comment:
I agree that PEP 570 would make the doc more explicit:

```suggestion
.. method:: kqueue.control(changelist, max_events, timeout=None, /) -> eventlist
```

Reply:
That's what I had in mind too.

Should we make this new change for 3.8+, but the previous suggested change for the 3.7 docs?